.reuse/dep5

```diff
@@ -1,11 +1,11 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: <YOUR-REPO-NAME>
-Upstream-Contact: <YOUR-CONTACT (MAIL ADDRESS ETC.)>
-Source: <https://github.com/sap/YOUR-REPO-NAME>
-Disclaimer: The code in this project may include calls to APIs ("API Calls") of
+Upstream-Name: kyma-environment-broker
+Upstream-Contact: team-gopher@sap.com
```

Comment:
should we add Phoenix DL list here?

DL Phoenix Team: DL_623B716E6A0D2D029907BEDC@global.corp.sap